README.md

```diff
@@ -84,7 +84,7 @@ configure your accessory using JSON:
 |`runningContactSensor`|Add a contact sensor to HomeKit that's _open_ when Roomba is running|
 |`binContactSensor`|Add a contact sensor to HomeKit that's _open_ when Roomba's bin is full|
 |`dockingContactSensor`|Add a contact sensor to HomeKit that's _open_ when Roomba is docking|
-|`noDockOnStop`|Do not send home when stopped|
+|`dockOnStop`|Sends roomba home to dock when stopped|
```

Comment:
I think this suggests we add another column with the default value, e.g.

```
|Key|Description|Default value|
...
|`dockOnStop`|Sends roomba home to dock when stopped|`true`|
```

I'm not sure what we put for the columns that don't have a default... just leave it blank?

Reply:
yes, I agree leave it blank if there is no default. additions made.